MediaBrowser.Model/Services/IHasRequestFilter.cs

```diff
@@ -7,18 +7,18 @@ namespace MediaBrowser.Model.Services
     public interface IHasRequestFilter
     {
         /// <summary>
-        /// Order in which Request Filters are executed.
+        /// Gets the order in which Request Filters are executed.
         /// &lt;0 Executed before global request filters
```

Comment:
```suggestion
        /// &lt;0 Executed before global request filters.
```

Might as well be consistent :)